BillingApp/BillingInfo/models.py

```diff
@@ -2,10 +2,10 @@
 from django.contrib.auth.models import User
 
 class BillInfo(models.Model):
-    #owner_name = models.ForeignKey(User)
-    bill_content = models.CharField(max_length = 50)
+    #owner = models.ForeignKey(User, on_delete = models.CASCADE)
+    content = models.CharField(max_length = 50)
     cost = models.IntegerField()
```

Comment:
maybe we should use `amount` instead of `cost`, because billing may be expenses or income.
Then we need a type for billing.

But it's ok we do this in later PR or have more discussion with this